super-team-picker/db/method.js

```diff
@@ -0,0 +1,39 @@
+module.exports = (cohort, method, quantity) => {
+    teams = {}
+    console.log(cohort.content.split(","))
+    console.log(method)
+    console.log(quantity)
```

Comment:
After you've finished debugging, removing the console.logs to prevent too many logs in the console.